src/lib/eliom_state.server.ml

```diff
@@ -94,7 +97,7 @@ let set_global_service_state_timeout
   let sitedata =
     Eliom_request_info.find_sitedata "set_global_service_timeout"
   in
-  let secure = Eliom_common.get_secure secure sitedata in
+  let secure = Eliom_common.get_secure secure sitedata () in
```

Comment:
@balat, no need to remove the commit. If my comment was not clear, I was suggesting `Eliom_common.get_secure ~secure_o:secure` (the compiler accepts the unlabeled version, and we have disabled the warning).